test/yamls/v1beta1/pipelineruns/pipelinerun-with-final-tasks.yaml

```diff
@@ -164,6 +164,18 @@ spec:
       workspaces:
         - name: source
           workspace: git-source
+    - name: display-git-commit
+      params:
+        - name: commit
+          value: $(tasks.clone-app-repo.results.commit)
+      taskSpec:
+        params:
+          - name: commit
+        steps:
+          - name: display-commit
+            image: alpine
+            script: |
+              echo $(params.commit)
```

Comment:
im confused, why does a yaml file with this name exist in 2 different places?

Reply:
Please refer to https://github.com/tektoncd/pipeline/pull/3564#issuecomment-736252906